setup.py

```diff
@@ -51,7 +51,7 @@
         'reportlab',  # windows binary pypi.python.org/pypi/reportlab
         'requests',
         'zeep',
-        'vatnumber',
+        'python-stdnum',
```

Comment:
The list in kinda sorted, so perhaps you can put it in the right spot (even if some are misordered)
Also need to check with @d-fence what needs to be updated in other setup files, if there is anything.

Reply:
python3-stdnum should be added in debian/control file in depends. Pay attention that they are also ordered.
If vatnumber is not used anymore, it should be removed from debian/control, setup.cfg, setup/package.dfdebian, setup/package.dffedora.
python3-stdnum already exists in setup.cfg, so no other changes are needed.

Reply:
@jbw-odoo

Reply:
@d-fence It's done. I also removed vatnumber/added stdnum in :
odoo/addons/point_of_sale/tools/posbox/overwrite_before_init/etc/init_posbox_image.sh
odoo/setup/win32/winpy_requirements.txt

Reply:
Thanks, LGTM from the packaging POV.